imagestreams/imagestreams.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: ruby
+  pretty_name: Ruby
+  sample_repo: "https://github.com/sclorg/ruby-ex.git"
+  category: builder
+  description: >-
+    Build and run Ruby APP_VERSION applications on DISTRO_NAME. For more information
+    about using this builder image, including OpenShift considerations, see
+    https://github.com/sclorg/s2i-ruby-container/blob/master/APP_VERSION/README.md.
+  imagestream_files:
+  - filename: ruby-centos.json
+    latest: "3.3-ubi9"
+    distros:
+      - name: UBI 7
+        app_versions: ["3.0"]
+
+      - name: UBI 8
+        app_versions: ["2.5", "3.1", "3.3"]
+
+      - name: UBI 9
+        app_versions: ["3.0", "3.1", "3.3"]
+
+  - filename: ruby-rhel.json
+    latest: "3.3-ubi9"
+    distros:
+      - name: UBI 7
+        app_versions: ["3.0"]
+
+      - name: UBI 8
+        app_versions: ["2.5", "3.1", "3.3"]
+
+      - name: UBI 9
+        app_versions: ["3.0", "3.1", "3.3"]
+
+  - filename: ruby-rhel-aarch64.json
+    latest: "3.3-ubi9"
+    distros:
+      - name: UBI 7
```

Comment:
ruby-rhel-aarch64.json should not contain imagestreams for RHEL7 afaik.

Reply:
fixed by https://github.com/sclorg/s2i-ruby-container/pull/547/commits/f5f53ec3ed26bfc1d68dbabfea1730298ab2dd78